README.md

```diff
@@ -270,16 +270,39 @@ the left of `"commit"`, it is given to the top-level parser.
 If you're writing a command-based application, you can use the [CommandRunner][]
 and [Command][] classes to help structure it. [CommandRunner][] has built-in
 support for dispatching to [Command][]s based on command-line arguments, as well
-as handling `--help` flags and invalid arguments. For example:
+as handling `--help` flags and invalid arguments.
+
+When using the [CommandRunner][] it replaces the [ArgParser][].
+
+In the following example we build a dart application called `dgit` that takes commands `commit` and `stash`.
+
+The [CommandRunner][] takes an 'executableName' which is ONLY used to generate the help message.
+
+> Note: for you final application to be called `dgit` rather than `dgit.dart` you would need to compile it using dart2native.
```

Comment:
Drop this line. `dart2native` is not the common approach for this. `pub global activate` is much more common - but in either case it doesn't need to be documented here.

Reply:
removed.